h2/src/test/org/h2/test/testScript.sql

```diff
@@ -10588,4 +10588,76 @@ create table z.z (id int);
 > ok
 
 drop schema z;
-> ok
\ No newline at end of file
+> ok
+
+----------------
+--- ENUM support
+----------------
+
+--- ENUM basic operations
```

Comment:
Is it possible to add a test for a custom type? Something like:

```sql
create type card_suit as enum('hearts', 'clubs', 'spades');
create table card (rank int, suit card_suit);
and etc...
```
I don't really know if it is necessary, but this is the use case I'm personally interested in and would love this to be working :)